layouts/community/ergodox/coderkun_neo3/keymap.c

```diff
@@ -0,0 +1,320 @@
+ * │       │ PgUp│ BSpc│  ↑  │ Del │ PgDn│     │     │     │     │  7  │  8  │  9  │  +  │   −   │
+ * ├───────┼─────┼─────┼─────╆─────╅─────┤     │     │     ├─────╆─────╅─────┼─────┼─────┼───────┤
+ * │       │ Home│  ←  │  ↓  │  →  │ End ├─────┤     ├─────┤     │  4  │  5  │  6  │  ,  │   .   │
+ * ├───────┼─────┼─────┼─────╄─────╃─────┤(TL3)│     │(TL2)├─────╄─────╃─────┼─────┼─────┼───────┤
+ * │       │ Esc │ Tab │ Ins │ Ret │ Undo│     │     │     │     │  1  │  2  │  3  │     │       │
+ * └─┬─────┼─────┼─────┼─────┼─────┼─────┴─────┘     └─────┴─────┼─────┼─────┼─────┼─────┼─────┬─┘
+ *   │     │     │(MO1)│     │(MO4)│                             │(MO4)│     │(MO1)│     │     │
+ *   └─────┴─────┴─────┴─────┴─────┘ ┌─────┬─────┐ ┌─────┬─────┐ └─────┴─────┴─────┴─────┴─────┘
+ *                                   │     │     │ │     │     │
+ *                             ┌─────┼─────┼─────┤ ├─────┼─────┼─────┐
+ *                             │     │     │     │ │     │     │     │
+ *                             │     │     ├─────┤ ├─────┤     │     │
+ *                             │     │     │     │ │     │     │     │
+ *                             └─────┴─────┴─────┘ └─────┴─────┴─────┘
+ */
+[NHL] = LAYOUT_ergodox(
+        // left hand
+        KC_NO,      KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,
+        KC_NO,      KC_PGUP,KC_BSPC,KC_UP,  KC_DEL, KC_PGDN,KC_NO,
+        KC_NO,      KC_HOME,KC_LEFT,KC_DOWN,KC_RGHT,KC_END,
+        KC_NO,      KC_ESC, KC_TAB, KC_INS, KC_ENT, KC_UNDO,KC_TRNS,
+        KC_NO,      KC_NO,  KC_TRNS,KC_NO,  KC_TRNS,
+                                                    KC_NO,      KC_NO,
+                                                                KC_NO,
+                                            KC_NO,  KC_NO,      KC_NO,
+        // right hand
+        KC_NO,      KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,  KC_NO,
+        KC_NO,      KC_NO,  KC_7,   KC_8,   KC_9,   DE_PLUS,DE_MINS,
+                    KC_NO,  KC_4,   KC_5,   KC_6,   KC_COMM,KC_DOT,
+        KC_TRNS,    KC_NO,  KC_1,   KC_2,   KC_3,   KC_NO,  KC_NO,
+                            KC_TRNS,KC_NO,  KC_TRNS,KC_NO,  KC_NO,
+        KC_NO,      KC_NO,
+        KC_NO,
+        KC_NO,      KC_NO,  KC_NO
+    )
+};
+
+const uint16_t PROGMEM fn_actions[] = {
+    [BASE] = ACTION_FUNCTION(BASE),
+    [FMU]  = ACTION_LAYER_TAP_TOGGLE(FMU),
+    [PMN]  = ACTION_FUNCTION(PMN),
+    [PMB]  = ACTION_FUNCTION(PMB),
+    [NHL]  = ACTION_LAYER_TAP_TOGGLE(NHL)
+};
+
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
+{
+    // MACRODOWN only works in this function
+    switch(id) {
+        case 0:
+            if (record->event.pressed) {
+                register_code(KC_RSFT);
+            }
+            else {
+                unregister_code(KC_RSFT);
+            }
+        break;
+    }
+
+    return MACRO_NONE;
+};
```

Comment:
This method of macros is deprecated, and should be updated to https://docs.qmk.fm/#/feature_macros for it to be accepted.

Reply:
Thanks for the link, I will look into it.

Reply:
I fixed this by removing the macro.